Rakefile

```diff
@@ -59,10 +59,19 @@ namespace :test do
 end
 
 namespace :lint do
+  task swift: ['run:swiftlint', 'run:swiftformat']
+
   desc 'Lints swift files'
-  task :swift do
-    sh 'mint run SwiftLint lint Sources Tests Example Package.swift --config script/lint/swiftlint.yml --strict'
-    sh 'mint run SwiftFormat Sources Tests Example Package.swift --config script/lint/airbnb.swiftformat --lint'
+  namespace :run do
```

Comment:
I think it's valid to have a namespace that's the same as a task name, so maybe we could just nest these under a swift namespace to make them more discoverable?

Reply:
aah good idea, didn't know that was allowed